src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -6803,6 +6803,7 @@ void MacroAssembler::verify_cross_modify_fence_not_required() {
 #endif
 
 void MacroAssembler::spin_wait() {
+  block_comment("spin_wait");
```

Comment:
You need something unique. e.g. 
```suggestion
  block_comment("spin_wait_ohthoo8H");
```

Reply:
Erm, I don't see why?

Reply:
But this is a generic macroAssembler block comment. It does not make sense to me to have a block comment that looks like a memory corrupted string, just to satisfy a single test. There should be a middle-ground here, e.g. `spin_wait {`, which looks reasonably enough as the block comment, and not anything else.

Reply:
Putting `_3_nop` might be convenient, but I think that would introduce too much hassle in macroAssembler? Just this looks okay to me:

```
;; spin_wait {
nop
nop
nop
;; }
```

Reply:
@theRealAph, are you okay with the latest variant Aleksey proposes?

Reply:
> But this is a generic macroAssembler block comment. It does not make sense to me to have a block comment that looks like a memory corrupted string, just to satisfy a single test. There should be a middle-ground here, e.g. `spin_wait {`, which looks reasonably enough as the block comment, and not anything else. Would also match nicely when we emit the closing `}` at the end of the instruction block.

OK

Reply:
I update the PR to use `spin_wait { ... }`